CHANGELOG.rst

```diff
@@ -23,6 +23,8 @@ Fixed
 
 - incorrect atomic mass unit for xTB calculations
 
+- Check of minimum distances from neighbours
+
```

Comment:
I'd suggest not to mention this in the change log, as the change is basically invisible for the users. (The only thing they would see, that they have to put the atoms even closer as so far, in order to trigger an error message from DFTB+).

Reply:
@aradi I removed the change log.

Reply:
This bug exists in version 21.1 and 21.2 (I did not check the version 20.2).

Reply:
The bug probably has been here for 10+ years. It will trigger for atoms at < 0.01 Bohr separation, so probably never manifested in real systems.